src/containers/exchange/components/OrderBook.js

```diff
@@ -41,12 +41,12 @@ const OrderItem = props => {
 
     const ret = [
         <Col style={{ padding: ".1em 0" }} width={3} key={`${order.direction}-amount`}>
-            {order.direction === TOKEN_SELL && <StyledSpan>{order.amount.toFixed(2)} A€</StyledSpan>}
+            {order.direction === TOKEN_SELL && <StyledSpan>{order.amount} A€</StyledSpan>}
```

Comment:
why not `toFixed(2)`?

![image](https://user-images.githubusercontent.com/32713470/55088750-94a5c680-50ac-11e9-8220-b93609169311.png)


```suggestion
            {order.direction === TOKEN_SELL && <StyledSpan>{order.amount.toFixed(2)} A€</StyledSpan>}
```

Reply:
because I decluttered the orders in store in [this commit](https://github.com/Augmint/augmint-web/pull/556/commits/5d3f342085d7314e5631b565ae71797263e86109), it was very convoluted. 
Basically no formatted data is stored in store for orders anymore. So amounts are with all significant decimals in store, formatting is up to the UI component. But AEUR has only 2 significant decimals which is already parsed so in [this line](https://github.com/Augmint/augmint-web/blob/5d3f342085d7314e5631b565ae71797263e86109/src/modules/ethereum/exchangeTransactions.js#L88)

Reply:
@szerintedmi are trailing zeroes still properly printed (as with `toFixed`)?

Reply:
no. do we need it? if so then @admrid 's suggested change makes sense

Reply:
Yes, it should be displayed consistently in the UI, but particularly important here to preserve column alignment. 

I have previously suggested introducing a `formatToken` method that all display should go through. I still do :)

Reply:
OK, I inclide @admrid 's change then, thanks

as for `formatToken` (and perhaps `formatEth` and percent ?) whoever will do it should have a look at `utils/converter.js`, someone started something already long back and it might have been already used some places?